Sources/Utilities/OneLoginAnalyticsService.swift

```diff
@@ -0,0 +1,27 @@
+import Foundation
+import Logging
+
+final class OneLoginAnalyticsService: AnalyticsService {
+    
+    var additionalParameters = [String: Any]()
+    
+    func grantAnalyticsPermission() {
+        // empty in lieu of analytics tickets
```

Comment:
maybe get a TODO with a ticket/epic number in it for these if you have it?

Reply:
don't think we have any yet unfortunately